chevah/compat
# [#2234] Check for SeImpersonatePrivilege when reporting Windows capabilities.

Scope
=====

When running SFTPPlus in debug without administrator privileges it's unable to authenticate OS accounts.

The reason for this is that without `SeImpersonatePrivilege` we cannot impersonate the user and access it's home folder.

For legacy reasons on Windows we were reporting this availability as always available.

Changes
=======

Fixed the code to check for the specified privilege when reporting local account impersonation capability.

How to try and test the changes
===============================

reviewers: @adiroiban 

Please check that changes make sense. Once this is in I will fix the documentation on the server to describe this scenario and what should be done.
